guide/contributing/developer-guide.md

```diff
@@ -60,12 +59,33 @@ ensure conventional commit messages, which supports our publishing workflow and
 You will not be able to make a commit until the precommit tasks pass. We also have a prepush hook to run a full build before pushing your code.
 
 Automated component versioning is facilitated by the [Conventional Commits specification](https://conventionalcommits.org/), so you must
-be deliberate when choosing the type of commit. For example, only use the "feat" type if you are adding a backwards-compatible feature
-to a component.
+be deliberate when choosing the type of commit.
+
+You can follow conventional commits like this:
+
+* For breaking changes, use the **feat** commit type with body text that begins with the phrase "BREAKING CHANGE"
+* For minor changes, use the **feat** commit type
+* For patches, use the **fix** commit type
+
+To view a guide on how TDS components are versioned, see our [FAQ](../faq.md#how-is-tds-versioned).
 
 ```bash
 # Stage your files and make a commit using commitizen
 yarn cz
+
+## At the `type` and `scope` prompts, you can press TAB to view options
```

Comment:
Please add rationale for the scope, and list commonly used cases.

Reply:
Done!